react-renderer/src/Constraints.ts

```diff
@@ -6,230 +6,230 @@ const TOL = 1e-4;
 const DEBUG_ENERGY = false;
 
 export const objDict = {
-  equal: (x: DiffVar, y: DiffVar) => squaredDifference(x, y),
```

Comment:
In this commit my IDE auto-indented the file, so the diff is misleading. Maybe split this commit so it doesn't pollute the blame history